keycloak/realms.go

```diff
@@ -0,0 +1,235 @@
+	VerifyEmail                         *bool                                  `json:"verifyEmail,omitempty"`
+	WaitIncrementSeconds                int                                    `json:"waitIncrementSeconds,omitempty"`
+}
+
+// AuthenticationFlowRepresentation for representing Flows
+type AuthenticationFlowRepresentation struct {
+	Alias                    string                                        `json:"alias,omitempty"`
+	AuthenticationExecutions []AuthenticationExecutionExportRepresentation `json:"authenticationExecutions,omitempty"`
+	BuiltIn                  *bool                                         `json:"builtIn,omitempty"`
+	Description              string                                        `json:"description,omitempty"`
+	ID                       string                                        `json:"id,omitempty"`
+	ProviderID               string                                        `json:"providerID,omitempty"`
+	TopLevel                 *bool                                         `json:"topLevel,omitempty"`
+}
+
+// AuthenticationExecutionExportRepresentation for Authenticator Execution
+type AuthenticationExecutionExportRepresentation struct {
+	Authenticator       string `json:"authenticator,omitempty"`
+	AuthenticatorConfig string `json:"authenticatorConfig,omitempty"`
+	AuthenticatorFlow   *bool  `json:"authenticatorFlow,omitempty"`
+	AutheticatorFlow    *bool  `json:"autheticatorFlow,omitempty"`
+	FlowAlias           string `json:"flowAlias,omitempty"`
+	Priority            int    `json:"priority,omitempty"`
+	Requirement         string `json:"requirement,omitempty"`
+	UserSetupAllowed    *bool  `json:"userSetupAllowed,omitempty"`
+}
+
+// AuthenticatorConfigRepresentation Authenticator Config
+type AuthenticatorConfigRepresentation struct {
+	Alias  string       `json:"alias,omitempty"`
+	Config AttributeMap `json:"config,omitempty"`
+	ID     string       `json:"id,omitempty"`
+}
+
+// ClientScopeRepresentation Client Scope
+type ClientScopeRepresentation struct {
+	Attributes      AttributeMap                   `json:"attributes,omitempty"`
+	Description     string                         `json:"description,omitempty"`
+	ID              string                         `json:"id,omitempty"`
+	Name            string                         `json:"name,omitempty"`
+	Protocol        string                         `json:"protocol,omitempty"`
+	ProtocolMappers []ProtocolMapperRepresentation `json:"protocolMappers,omitempty"`
+}
+
+// IdentityProviderMapperRepresentation Identity Provider Mapper
+type IdentityProviderMapperRepresentation struct {
+	Config                 AttributeMap `json:"config,ommitempty,omitempty"`
+	ID                     string       `json:"id,ommitempty,omitempty"`
+	IdentityProviderAlias  string       `json:"identityProviderAlias,ommitempty,omitempty"`
+	IdentityProviderMapper string       `json:"identityProviderMapper,ommitempty,omitempty"`
```

Comment:
Could we remove all instances of `,ommitempty`? Looks like a typo on original go generate.

Reply:
You want me to remove `,omitempty` for all structs or just the ones I added?

Reply:
oh, missed that. Thanks!